### ISSUE TITLE
When a map is created with no controls, the Toggle Controls context menu item should be disabled

### ISSUE THREAD

**Issue by prushforth** [Maps4HTML/Web-Map-Custom-Element#696]
Or not, but it currently doesn't do anything when the `controls` attribute is omitted, despite being enabled.

**Comment by AliyanH** [comment]
Need to add implementation/testing for web-map.js
